steering_controllers_library/src/steering_odometry.cpp

```diff
@@ -148,7 +148,7 @@ bool SteeringOdometry::update_from_velocity(
   steer_pos_ = (right_steer_pos + left_steer_pos) * 0.5;
   double linear_velocity =
     (right_traction_wheel_vel + left_traction_wheel_vel) * wheel_radius_ * 0.5;
-  const double angular_velocity = steer_pos_ * linear_velocity / wheelbase_;
+  const double angular_velocity = tan(steer_pos_) * linear_velocity / wheelbase_;
```

Comment:
```suggestion
  const double angular_velocity = std::tan(steer_pos_) * linear_velocity / wheelbase_;
```

Reply:
I did that already in the other steering library PR -> we well have merge conflicts now ;)